test/utest/src/cuter_metrics_tests.erl

```diff
@@ -0,0 +1,49 @@
+%% -*- erlang-indent-level: 2 -*-
+%%------------------------------------------------------------------------------
+-module(cuter_metrics_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-spec test() -> 'ok' | {'error', term()}.  %% This should be provided by EUnit
+
+-spec distribution_metric_test() -> ok.
+distribution_metric_test() ->
+  Name = some_metric,
+  ok = cuter_metrics:start(),
+  ok = cuter_metrics:define_distribution_metric(Name),
+  ok = cuter_metrics:measure_distribution(Name, 42),
```

Comment:
These 6 lines can become just one:
```erlang
   lists:foreach(fun (N) -> cuter_metrics:measure_distribution(Name, N) end, [42, 1, 42, 23, 1, 42]),
```
A similar change can occur in the next function (with xxx, etc.) and there we can actually use the same list in both the call to `foreach` and in the `assertEqual`.

Reply:
Done.